debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlJdbcContext.java

```diff
@@ -47,7 +48,8 @@ public MySqlJdbcContext(Configuration config) {
         // to give us better JDBC database metadata behavior, including using UTF-8 for the client-side character encoding
         // per https://dev.mysql.com/doc/connector-j/5.1/en/connector-j-reference-charsets.html
         boolean useSSL = sslModeEnabled();
-        Configuration jdbcConfig = config.subset("database.", true)
+        Configuration jdbcConfig = config.filter(x -> !(x.startsWith(DatabaseHistory.CONFIGURATION_FIELD_PREFIX_STRING) || x.equals(MySqlConnectorConfig.DATABASE_HISTORY.name())))
```

Comment:
Wouldn't

    Configuration jdbcConfig = config.filter(x -> !x.startsWith(MySqlConnectorConfig.DATABASE_HISTORY.name()))

be enough?

Reply:
This is intentinal to protect against a situation when a new JDBC connection string parameter intoruduced in future would be named history<XXX>.

Reply:
Ok :)